### PR TITLE
Update config.yml

### PR DESCRIPTION
**Pull request by ricardogobbosouza**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,16 +1,15 @@
...
-      - image: banian/node
+      - image: circleci/node
     steps:
       # Checkout repository
       - checkout
 
       # Restore cache
       - restore_cache:
-          key: yarn-{{ checksum "yarn.lock" }}
+          key: yarn-cache-{{ checksum "yarn.lock" }}
```
Comment (NicoPennec) [inline .circleci/config.yml]: @ricardogobbosouza  can you explain this update, please?

File: .circleci/config.yml [hunk 2]
```diff
@@ -19,14 +18,19 @@ jobs:
...
+      # Lint
+      - run:
+          name: Lint
+          command: yarn lint
+
+      # Tests
       - run:
           name: Tests
-          command: yarn test
+          command: yarn jest
```
Comment (NicoPennec) [inline .circleci/config.yml]: you should call `yarn unit`

File: .circleci/config.yml [hunk 3]
```diff
@@ -19,14 +18,19 @@ jobs:
...
 
-      # Test
+      # Lint
+      - run:
+          name: Lint
+          command: yarn lint
+
+      # Tests
       - run:
           name: Tests
```
Comment (NicoPennec) [inline .circleci/config.yml]: if we call "yarn unit", it should be clearer to rename it by "Unit". what do you think?

File: .circleci/config.yml [hunk 4]
```diff
@@ -1,16 +1,15 @@
 version: 2
 jobs:
   build:
-    working_directory: /usr/src/app
```
Comment (NicoPennec) [inline .circleci/config.yml]: why remove this line?

eg. https://github.com/nuxt/nuxt.js/blob/dev/.circleci/config.yml#L4

File: .circleci/config.yml [hunk 5]
```diff
@@ -1,16 +1,15 @@
...
     docker:
-      - image: banian/node
+      - image: circleci/node
```
Comment (NicoPennec) [inline .circleci/config.yml]: 👍 

to be aligned with Nuxt, it must be better to set `circleci/node:latest` 

see https://github.com/nuxt/nuxt.js/blob/dev/.circleci/config.yml#L6

**Comment by ricardogobbosouza** [comment]
@NicoPennec I'm releasing an update PR in the modules, to follow the same model as https://github.com/nuxt-community/module-template, I have not done this here, ignore this PR

**Pull request closed without merging**